### PR TITLE
small update to cigar docs

### PR DESCRIPTION
**Pull request by craiglowe**
# Description
Adding a couple comments to the cigar package.

## Relevant Links

- [Display Text](https://www.vertgenlab.org/)

### Checklist before requesting a review

- [ ] I performed a self-review of my code
- [ ] If it this a core feature, I have added thorough tests
- [ ] The command `go fmt` was used on all files included

### Testing

None

### Screenshots & Media

None

### Edge cases / Breaking Changes / Known Issues

None

### CODE REVIEW

File: cigar/tools.go [hunk 1]
```diff
@@ -1,5 +1,8 @@
 package cigar
 
+// AddCigar ads a cigar struct to the end of a slice, but is smart about checking
```
Comment (cfauci) [inline cigar/tools.go:3]: misspelling "ads"

Comment (craiglowe) [reply cigar/tools.go]: Thank you for catching this.  I just pushed a fix

**Review by rimangan — APPROVED** [review]

**Review by cfauci — APPROVED** [review]
